eds4jinja2/__init__.py

```diff
@@ -9,4 +9,15 @@
 
 # The format of the __version__ line is matched by a regex in setup.py and /docs/conf.py
 __version__ = "0.1.19"
-__date__ = "2020-08-14"
\ No newline at end of file
+__date__ = "2020-08-14"
+
+import logging
+
+# hard coding the log level
```

Comment:
what do you mean by "hard coding"?

Reply:
well, ideally this shall be in an .ini/.cfg file